Makefile

```diff
@@ -36,6 +37,7 @@ lint:
 	black --check --diff $(PY_FILES)
 	isort --check --diff $(PY_FILES)
 	flake8 $(PY_FILES)
+	mypy --show-error-codes $(MYPY_FILES)
```

Comment:
Does this pass CI? I could imagine this causes it to fail for a while. Maybe to keep lint passing, add another make target?

Reply:
I only added `dependencies.py` for now.